app/code/community/WindsorCircle/Analytics/etc/config.xml

```diff
@@ -313,6 +313,10 @@ updated_at
 total_orders
 total_spent
 </customer_traits>
+                <product_properties>
+cost
```

Comment:
so maybe I'm crazy, but it looks like you are adding these, not removing them?  Unless this is a blacklist?  But it doesn't look like that